packages/notification-center/src/components/notification-center/components/Loader.tsx

```diff
@@ -1,9 +1,7 @@
 import React from 'react';
 import { Loader as MantineLoader } from '@mantine/core';
 import { cx, css } from '@emotion/css';
-import chroma from 'chroma-js';
```

Comment:
removed `chroma-js` it was used to translate the `linear-gradient()` to an average hex color, so instead I've calculated that color for the NC and used it in the theme config... the weight of that lib was ~18kB gzip, ~45kB unpacked... no sense to use library for that